### ISSUE TITLE
Crash Report: IndexOutOfBoundsException KiwixMobileActivity.onResume (KiwixMobileActivity.java:1263)

### ISSUE THREAD

**Issue by macgills** [kiwix/kiwix-android#1261]
**Describe the bug**
```
java.lang.RuntimeException: 

 
  at android.app.ActivityThread.performResumeActivity (ActivityThread.java:4290)

 
  at android.app.ActivityThread.handleResumeActivity (ActivityThread.java:4330)

 
  at android.app.servertransaction.ResumeActivityItem.execute (ResumeActivityItem.java:51)

 
  at android.app.servertransaction.TransactionExecutor.executeLifecycleState (TransactionExecutor.java:145)

 
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:70)

 
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2123)

 
  at android.os.Handler.dispatchMessage (Handler.java:109)

 
  at android.os.Looper.loop (Looper.java:207)

 
  at android.app.ActivityThread.main (ActivityThread.java:7470)

 
  at java.lang.reflect.Method.invoke (Native Method)

 
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:524)

 
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:958)
Caused by: java.lang.IndexOutOfBoundsException: 

 
  at java.util.ArrayList.get (ArrayList.java:437)

 
  at org.kiwix.kiwixmobile.KiwixMobileActivity.onResume (KiwixMobileActivity.java:1263)

 
  at android.app.Instrumentation.callActivityOnResume (Instrumentation.java:1456)

 
  at android.app.Activity.performResume (Activity.java:7592)

 
  at android.app.ActivityThread.performResumeActivity (ActivityThread.java:4272)
```

**Comment by macgills** [comment]
This was already fixed

**Comment by julianharty** [comment]
This bug exists in the current (2.5.x) releases and occurs many times a day (1899 times in 60 days). @abdulwd fixed this issue in the 3.0 codebase, please could we back-port it to 2.5.x and make a bug-fix release so that we reduce the crash rate for the app in production?

```
![IndexOutOfBoundsException - Google Play Console](https://user-images.githubusercontent.com/785891/63209854-2c4c9580-c0de-11e9-85f2-7ebca4b11abb.png)

Today, 7:58 AM on app version 6220502
Samsung Galaxy A50 (a50), Android 9
Report 1
java.lang.RuntimeException: 
  at android.app.ActivityThread.performResumeActivity (ActivityThread.java:4015)
  at android.app.ActivityThread.handleResumeActivity (ActivityThread.java:4047)
  at android.app.servertransaction.ResumeActivityItem.execute (ResumeActivityItem.java:51)
  at android.app.servertransaction.TransactionExecutor.executeLifecycleState (TransactionExecutor.java:145)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:70)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1955)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:214)
  at android.app.ActivityThread.main (ActivityThread.java:7058)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:965)
Caused by: java.lang.IndexOutOfBoundsException: 
  at java.util.ArrayList.get (ArrayList.java:437)
  at org.kiwix.kiwixmobile.KiwixMobileActivity.onResume (KiwixMobileActivity.java:1263)
  at android.app.Instrumentation.callActivityOnResume (Instrumentation.java:1416)
  at android.app.Activity.performResume (Activity.java:7558)
  at android.app.ActivityThread.performResumeActivity (ActivityThread.java:4007)
```

**Comment by julianharty** [comment]
@kelson42 Here's the issue I mentioned which I hope would be worth deploying a fix for in 2.5.x to reduce the crash rate for our users (and improve our ranking in Google Play).
